ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,156 @@
+    Navigate To Page    Settings    Accelerator profiles
+    Click On Edit Accelerator Profile    ${ACC_DISPLAY_NAME}3
+    Delete Accelerator Profile Tolerations    ${ACC_DISPLAY_NAME}3
+    In The Accelerator Profiles Grid There Is An Accelerator Profile With Name   ${ACC_DISPLAY_NAME}3
+    Verify Accelerator Profile Has No Tolerations Via CLI   ${ACC_DISPLAY_NAME}3
+
+Disable an Accelerator Profile From The Accelerator Profile Edit View
+    [Documentation]    Disable an An accelerator profile from the Edit Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}4
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Click On Edit Accelerator Profile    ${ACC_DISPLAY_NAME}4
+    Modify The Accelerator Profile    original_display_name=${ACC_DISPLAY_NAME}4    enabled=False
+    In The Accelerator Profiles Grid There Is An Accelerator Profile With Name   ${ACC_DISPLAY_NAME}4
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}4    enabled=False
+
+Enable an Accelerator Profile From The Accelerator Profile Edit View
+    [Documentation]    Enable an An accelerator profile from the Edit Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}5
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Click On Edit Accelerator Profile    ${ACC_DISPLAY_NAME}5
+    Modify The Accelerator Profile    original_display_name=${ACC_DISPLAY_NAME}5    enabled=True
+    In The Accelerator Profiles Grid There Is An Accelerator Profile With Name   ${ACC_DISPLAY_NAME}5
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}5    enabled=True
+
+Disable an Accelerator Profile From The Grid of the Accelerator Profile View
+    [Documentation]    Disable an An accelerator profile from the Edit Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}6
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Search for accelerator profile in grid    ${ACC_DISPLAY_NAME}6
+    Disable Accelerator Profile    ${ACC_NAME}6
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}6    enabled=False
+
+Enable an Accelerator Profile From The Grid of the Accelerator Profile View
+    [Documentation]    Enable an An accelerator profile from the Edit Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}7
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Search for accelerator profile in grid    ${ACC_DISPLAY_NAME}7
+    Click On Enable Accelerator Profile    ${ACC_NAME}7
```

Comment:
what about adding the check for disabling the profile as well inside this test case?